pallets/avn-proxy/src/tests/mock.rs

```diff
@@ -293,6 +293,21 @@ pub fn proxy_event_emitted(
     })
 }
 
+pub fn proxy_failed_inner_dispatch_event_emitted(
```

Comment:
This function name no longer matches the event name

Reply:
updated